tests/providers/google/cloud/operators/test_dataproc.py

```diff
@@ -419,6 +428,24 @@ def test_execute(self, mock_hook):
 
 
 class TestDataprocSubmitJobOperator(unittest.TestCase):
+
+    def setUp(self):
+        self.dagbag = DagBag(
+            dag_folder='/dev/null', include_examples=False
+        )
+        self.dag = DAG(TEST_DAG_ID, default_args={
+            'owner': 'airflow',
+            'start_date': DEFAULT_DATE
+        })
+        self.mock_context = MagicMock()
+
+    def tearDown(self):
+        session = Session()
+        session.query(TaskInstance).filter_by(
+            dag_id=TEST_DAG_ID).delete()
+        session.commit()
+        session.close()
```

Comment:
Let's use `setUpClass` so we do this logic once

Reply:
And we can use `tests.tests_utils.db.clear_db_runs` and `clear_db_xcom` as we push something to XCom

Reply:
Have made those changes. One question though, I've added the tests in DataProcSubmiSparkOperator(DataprocJobBaseOperator). Should I add the tests for other subclass operators as well?

Reply:
Those operators are depecated so it's ok if we test with only one. But please add tests for `DataprocSubmitJobOperator`

Reply:
Added already.
https://github.com/apache/airflow/pull/10343/files#diff-68c5fc800ea76a624d56445376a5744fR485